nuxt.config.js

```diff
@@ -74,4 +74,12 @@ export default {
       scss: { sourceMap: false },
     },
   },
+
+  vue: {
+    // TODO REMOVE THIS
+    config: {
+      productionTip: true,
+      devtools: false,
+    },
+  },
```

Comment:
Remember to remove this!

Reply:
I had the extension to remind it to me 😋, but thanks for the reminder anyway!
Removed! 🗑️